.github/workflows/tests.yml

```diff
@@ -242,20 +289,10 @@ jobs:
         with:
           go-version: ${{ matrix.go }}
 
-      - name: Check compatibility with min Go version
-        run: |
-          set -eux
-          GOMIN="$(sed -n 's/^GOMIN=\([0-9.]\+\)$/\1/p' Makefile)"
-          go mod tidy -go="${GOMIN}"
-
-          DOC_GOMIN="$(sed -n 's/^LXD requires Go \([0-9.]\+\) .*/\1/p' doc/requirements.md)"
-          [ "${GOMIN}" = "${DOC_GOMIN}" ]
-
       - name: Install dependencies
         run: |
           set -eux
           sudo add-apt-repository ppa:ubuntu-lxc/daily -y --no-update
-          sudo add-apt-repository ppa:dqlite/dev -y --no-update
```

Comment:
If we are not compiling LXD here, do we need the `ubuntu-lxc/daily` PPA and the `liblxc-dev` and other `-dev` packages?

Reply:
for now we do as LXD links against it dynamically